OSX : permitted only in C++11delegating constructors are permitted only in C++11

My problem is below. How to fix this?

mbpros:build mbpros$ make
Scanning dependencies of target generate_resources_tool
[  2%] Building CXX object CMakeFiles/generate_resources_tool.dir/tools/generate_resources.cpp.o
[  5%] Linking CXX executable bin/generate_resources_tool
[  5%] Built target generate_resources_tool
[  7%] Generating resources.inc.h
Scanning dependencies of target freenect2
[ 10%] Building CXX object CMakeFiles/freenect2.dir/src/transfer_pool.cpp.o
[ 12%] Building CXX object CMakeFiles/freenect2.dir/src/event_loop.cpp.o
[ 15%] Building CXX object CMakeFiles/freenect2.dir/src/usb_control.cpp.o
[ 17%] Building CXX object CMakeFiles/freenect2.dir/src/allocator.cpp.o
[ 20%] Building CXX object CMakeFiles/freenect2.dir/src/frame_listener_impl.cpp.o
[ 22%] Building CXX object CMakeFiles/freenect2.dir/src/packet_pipeline.cpp.o
[ 25%] Building CXX object CMakeFiles/freenect2.dir/src/rgb_packet_stream_parser.cpp.o
[ 27%] Building CXX object CMakeFiles/freenect2.dir/src/rgb_packet_processor.cpp.o
[ 30%] Building CXX object CMakeFiles/freenect2.dir/src/depth_packet_stream_parser.cpp.o
[ 32%] Building CXX object CMakeFiles/freenect2.dir/src/depth_packet_processor.cpp.o
[ 35%] Building CXX object CMakeFiles/freenect2.dir/src/cpu_depth_packet_processor.cpp.o
[ 37%] Building CXX object CMakeFiles/freenect2.dir/src/resource.cpp.o
[ 40%] Building CXX object CMakeFiles/freenect2.dir/src/command_transaction.cpp.o
In file included from /Users/mbpros/Desktop/libfreenect2/src/command_transaction.cpp:29:
In file included from /Users/mbpros/Desktop/libfreenect2/include/internal/libfreenect2/protocol/command_transaction.h:32:
/Users/mbpros/Desktop/libfreenect2/include/internal/libfreenect2/protocol/command.h:245:7: error: delegating constructors are permitted only in C++11
    : ColorSettingCommand(cmd, float_bytes_to_uint32_(value))
      ^~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/freenect2.dir/src/command_transaction.cpp.o] Error 1
make[1]: *** [CMakeFiles/freenect2.dir/all] Error 2
make: *** [all] Error 2
mbpros:build mbpros$ delegating constructors are permitted only in C++11delegating constructors are permitted only in C++11delegating constructors are permitted only in C++11delegating constructors are permitted only in C++11delegating constructors are permitted only in C++11

--- a/include/internal/libfreenect2/protocol/command.h
+++ b/include/internal/libfreenect2/protocol/command.h
@@ -242,16 +242,14 @@ struct ColorSettingCommand : public Command<KCMD_RGB_SETTING, ColorSettingRespon
 
   // Could overload ctor to ease usage for float-valued settings.
   ColorSettingCommand(ColorSettingCommandType cmd, float value)
-    : ColorSettingCommand(cmd, float_bytes_to_uint32_(value))
+    : Command<KCMD_RGB_SETTING, ColorSettingResponseSize, ColorSettingResponseSize, 4>(0)
   {
-  }
-
-private:
-  static uint32_t float_bytes_to_uint32_(float value)
-  {
-    uint32_t out;
-    memcpy(&out, &value, sizeof(out));
-    return out;
+    this->data_.parameters[0] = 1;
+    this->data_.parameters[1] = 0;
+    this->data_.parameters[2] = (uint32_t)cmd;
+    uint32_t value2;
+    memcpy(&value2, &value, sizeof(value2));
+    this->data_.parameters[3] = value2;
   }
 };
 

